manticore/native/cpu/abstractcpu.py

```diff
@@ -999,7 +1012,7 @@ def execute(self):
                     )
                     self.backup_emulate(insn)
         except (Interruption, Syscall) as e:
-            e.on_handled = lambda: self._publish_instruction_as_executed(insn)
+            self._delayed_event = True
```

Comment:
So `on_handled` never gets called, eh? Could there be another bug? Seems like it's supposed to be handled right here: https://github.com/trailofbits/manticore/blob/master/manticore/platforms/linux.py#L2428-L2429

Reply:
When the syscall wants to fork or raises the on_handled gets ignored?
![image](https://user-images.githubusercontent.com/1017522/66157979-4a714500-e5fb-11e9-9066-ba4efa514db8.png)